tracer.go

```diff
@@ -5,7 +5,7 @@ import "time"
 // Tracer is a simple, thin interface for Span creation.
 //
 // A straightforward implementation is available via the
-// `opentracing/basictracer` package's `standardtracer.New()'.
+// `opentracing/basictracer-go` package's `standardtracer.New()'.
```

Comment:
In the sense of `package basictracer`, yes... in the sense of the pathname that we have here (or the github repo), no. No strong feelings but wanted to explain the reasoning.